### PR TITLE
Install dependencies in a way that doesn't confuse fabricate

### PR DESCRIPTION
**Pull request by Changaco**
Using `pip install --editable .` probably wasn't a good idea, fabricate was rerunning it every time.

### CODE REVIEW

**Comment by Changaco** [comment]
Ready for review.

File: setup.py [hunk 1]
```diff
@@ -32,6 +30,6 @@
      , version = version
      , zip_safe = False
      , package_data = {'aspen': ['request_processor/mime.types']}
-     , install_requires = ASPEN_DEPS
-     , tests_require = TEST_DEPS
+     , install_requires = open('requirements.txt').read()
+     , tests_require = open('tests/requirements.txt').read()
```
Comment (chadwhitacre) [inline setup.py]: What's the effect of this on `sdist`s of this package? Do we need to include the requirements files in `MANIFEST.in` or whatevs?

Comment (Changaco) [inline setup.py]: I hadn't thought of that, but it looks like they're being added to the `sdist` tarball automatically somehow. However I've found an easier solution that doesn't rely on magic, so I've forced-pushed it here. Previous HEAD was 7c4f8afdcb590f85e91d1a873b7a742156f118ac.

**Comment by Changaco** [comment]
CI is green.

**Comment by Changaco** [comment]
There's still one problem here: running `python build.py test` on a clean copy fails because `py.test` doesn't see the `aspen` module since we don't install anymore and it's not in the `PYTHONPATH`.

**Comment by Changaco** [comment]
Problem fixed, using `pip show` to detect if the package is already installed.

I've squashed everything into a single commit, previous HEAD was c5e2da8ed3cba5e71aec94f8544ae7652ebb19e0.

CI is green. Ready for review.

**Pull request merged**